src/dmd/glue.d

```diff
@@ -397,8 +397,10 @@ void genObjFile(Module m, bool multiobj)
         m.covb = cast(uint *)calloc((m.numlines + 32) / 32, (*m.covb).sizeof);
     }
 
-    foreach (member; *m.members)
+    const memberSize = m.members.dim;
+    for (int i = 0; i < memberSize; i++)
```

Comment:
Can’t a foreach range loop be used?

Reply:
Not 100% sure, but I think all `foreach` AST lowerings cache the range, and we need the bounds to be re-evaluated on every iteration.

Reply:
@ZombineDev is right.

Reply:
Use `m.members.dim` not `memberSize` for the same reason.

Reply:
Done